.changeset/fluffy-actors-listen.md

```diff
@@ -0,0 +1,26 @@
+---
+'@leafygreen-ui/text-input': major
+---
+
+[LG-4143](https://jira.mongodb.org/browse/LG-4143)
+
+Disabled `TextInput` component no longer renders the `disabled` attribute and instead relies on `aria-disabled`.
+
+This change is made to ensure that disabled components are still focusable to users using keyboard navigation.
+
+For more on `aria-disabled` see the [documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled)
```

Comment:
@shaneeza we should get this in before yours does, and then your test harness should reflect this change.

Reply:
I don't think this should be an issue. The test harness is already querying for `aria-disabled`

Reply:
Oh! Incredible.